pyclibrary/headers/update_WinDefs.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright 2015 by PyCLibrary Authors, see AUTHORS for more details.
+#
+# Distributed under the terms of the MIT/X11 license.
+#
+# The full license is in the file LICENCE, distributed with this software.
+# -----------------------------------------------------------------------------
+"""
+Windows headers are not included in this distribution due to possible
+copyright issues. Thus there is the cache file 'WinDefs.cache' here
+which encapsulate definitions pulled from several headers included
+in Visual Studio
+
+This script updatthe cache fileche.
+It has to be run on one of the following conditions:
+* a another Visual C/C++ compiler version is used
+* the object model of the parser was updated.
+"""
+from __future__ import (division, unicode_literals, print_function,
+                        absolute_import)
+from pyclibrary.c_parser import win_defs
+from pyclibrary.utils import add_header_locations
+
+SDK_DIR = r'c:\program files\microsoft sdks\windows\v6.0a\include'
```

Comment:
Unfortunately this is complicated, as every version has a different standard installaton path and the user can even install it into his own custom directory. Thus it would be necessary to check the windows registry. Could be done without too much effort (I think ~15 lines of code), but as this is no production code, I don't think it is worth doing the job...